.circleci/config.yml

```diff
@@ -172,70 +126,43 @@ jobs:
 workflows:
   version: 2
 
-  # The main workflow responsible for triggering all other workflows
-  # in which changes are detected.
-  ci:
-    when: << pipeline.parameters.trigger >>
-    jobs:
-      - trigger-workflows
-
-  # Trigger tag build when tag and not already building a specific package workflow
-  tag:
-    when:
-      and:
-        - << pipeline.git.tag >>
-        - not: << pipeline.parameters.wpe-headless-plugin >>
-    jobs:
-      - trigger-tag:
-          filters:
-            tags:
-              only: /^plugin\/wpe-headless\/\S+/
-
   # Workflows defined for each package.
-  # By default circleci doesn't trigger on a tag event. Add the filter for tags so that all jobs in the workflow will run also when a tag is present.
+  # tag example for deploying an update for wpe-headless: plugin/wpe-headless/1.0.0
   plugin-wpe-headless:
-    when: << pipeline.parameters.wpe-headless-plugin >>
     jobs:
-      - checkout:
-          filters:
-            tags:
-              only: /.*/
```

Comment:
You need these filters for any tag that way the jobs will run without a tag being specified. Ie, when a PR on a branch is created.  As well as when the actual tag is present.